features/gm.test.ts

```diff
@@ -146,11 +146,11 @@ Deno.test("should get the correct gm type", async () => {
     const creation = gm_A.createGroupChat();
     {
         // message
-        const messageEvent = await gm_A.prepareGroupMessageEvent(creation.groupKey.publicKey, "hello");
+        const messageEvent = await gm_A.prepareGroupMessageEvent(creation.groupKey.publicKey, "hello", []);
```

Comment:
A better API should be
```ts
prepareGroupMessageEvent(
  receiver_pubkey,
  {
     type: "text" | "image",
     content: string
  }
)
```

Reply:
We should hide the concept of nostr tags on surface level API because we might migrate away from NIP event definition in the future if Nostr can't keep up with DM needs.